src/generator/templates/Cargo.toml.mako

```diff
@@ -32,7 +32,7 @@ clap = "2"
 http-body-util = "0.1"
 % endif
 hyper = "1"
-hyper-rustls = "0.27"
+hyper-rustls = { version = "0.27", default-features = false }
```

Comment:
This is something extra. `hyper-rustls` has several default choices, which some might not like.

Reply:
It looks like now [the legacy support](https://github.com/Byron/google-apis-rs/actions/runs/11277500609/job/31363793254?pr=527#step:4:587) is also gone maybe?